cli/tests/pcluster/templates/test_cluster_stack/test_compute_launch_template_properties/cluster-using-flexible-instance-types.yaml

```diff
@@ -23,3 +23,6 @@ Scheduling:
     Networking:
       SubnetIds:
       - subnet-12345678
+{% if disable_sudo_access_default_user == True %}
```

Comment:
Why should we wrap the property within the if statement?
Wouldn't it be enough to do as follows?

```
DisableSudoAccessForDefaultUser: {{ disable_sudo_access_default_user }}
```